frontend/website/src/Svg.re

```diff
@@ -19,13 +19,17 @@ let make = (~link, ~dims, ~inline=false, ~className="", _children) => {
         Node.Fs.readFileAsUtf8Sync(
           String.sub(link, 1, String.length(link) - 1),
         );
-      <div dangerouslySetInnerHTML={"__html": content} />;
+      <div dangerouslySetInnerHTML={"__html": content} alt />;
     } else {
       <object
         data=link
         type_="image/svg+xml"
         width={Js.Float.toString(Size.remX(dims)) ++ "rem"}
         height={Js.Float.toString(Size.remY(dims)) ++ "rem"}
+        role={String.length(alt) == 0 ? "presentation" : "img"}
```

Comment:
maybe we should use a variant for `alt` instead of hacking empty string to mean something special (I was curious earlier why you were using empty for things)

Suggestion:

```
`Presentation
`Description("foo")
```

Reply:
empty string is a meaningful value for alt. It means you explicitly meant to put no description.
We could do this too though, I guess

Reply:
That's fair, let's keep it then